### PR TITLE
Add app header

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v3/src/components/shared/Logo/ProtocolTempus.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { FC } from 'react';
+import LogoProps from './LogoProps';
+import withLogo from './withLogo';
+
+const TokenDAI: FC<LogoProps> = () => (
```
Comment (ShawTim) [inline packages/tempus-client_v3/src/components/shared/Logo/ProtocolTempus.tsx:5]: `TokenDAI` should be `ProtocolTempus`, minor though

**Comment by ShawTim** [comment]
i'm not sure whether we should put the `Tempus Logo` under `shared/Logo` becoz it's not in 1:1 ratio. the `ProtocolTempus` should be expected to be used as a token icons like others but it gets a 120x30 size.
i think we should just put this SVG under `Navbar` folder since `Navbar` is the only consumer.
wdyt?

**Comment by lazarmitic** [comment]
Makes sense, I will move Tempus logo under Navbar directory.

**Comment by lazarmitic** [comment]
Fixed

**Review by ShawTim — APPROVED** [review]

**Pull request merged**